src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs

```diff
@@ -136,6 +145,136 @@ public RestoreCommand(IApplicationEnvironment env)
             }
         }
 
+        private async Task<bool> RestoreForInstall(string packagesDirectory)
```

Comment:
No, it was not. Also, it makes sense to have it as a different `Restore` because there is no project in the context of install